app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
-class ApplicationMailer < ActionMailer::Base
```

Comment:
We are not sending emails to users in our app at the moment so we comment out this functionaility